SI/Sem.04/Pract.04/Task15.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+using namespace std;
+bool isPrime(int num)
+{
+	int squreRoot = sqrt(num);
+	if (num <= 1)
+	{
+		return false;
+	}
+	bool isPrime = true;
+	for (int i = 2; i <= squreRoot; i++)
+	{
+		if (num % i == 0)
+		{
+			isPrime = false;
+			break;
+		}
+	}
+	return isPrime;
+}
+void factorization(int num)
+{
+	if (isPrime(num))
+	{
+		cout << "The number is prime and can't be factorise!";
+		return;
+	}
+	for (int i = 2; i <= num; i++)
```

Comment:
You should loop to the square root of the number
Also you can optimize it a bit for the even numbers